include/arc_utilities/geometry_msgs_builders.hpp

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <geometry_msgs/Pose.h>
+#include <geometry_msgs/Transform.h>
+
+namespace arc_utilities
```

Comment:
I'm hesitant to include too many types of messages in single file, if only to avoid file bloat. I might even suggest moving these into a sub-namespace as it's not unreasonable to have multiple libraries that define a "Point" or "Vector3". (Though most libraries would already have proper constructors.)

Reply:
:+1: new namespace
What about a subdirectory structure? We have a lot of files in the main directory.
`ros_msgs_builders.hpp` can just be:
```
#include arc_utilities/ros_msg_builders/geometry_msg_builders.hpp
#include arc_utilities/ros_msg_builders/color_msg_builders.hpp
```

Reply:
I'll leave it to the next person to move the files into a separate folder once there would be more than 1 file in that folder.